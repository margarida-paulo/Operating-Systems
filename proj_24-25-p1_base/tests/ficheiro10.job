SHOW
SHOW
DELETE [Natalie,Oscar]
SHOW