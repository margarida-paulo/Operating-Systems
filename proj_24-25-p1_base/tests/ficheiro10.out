(Anderson,Moura)
(Alexandra,Barros)
(André,Monteiro)
(Amanda,Santos)
(Alexandre,Rodrigues)
(Ana,Martins)
(Adriana,Lopes)
(Alberto,Pereira)
(Bruno,Neves)
(Beatriz,Moreira)
(Catarina,Castro)
(Carolina,Bastos)
(Cláudia,Barbosa)
(Carla,Lopes)
(Camila,Ribeiro)
(Clara,Sousa)
(Cristina,Sousa)
(Carlos,Ramos)
(Daniela,Lima)
(Douglas,Ramos)
(Diego,Marques)
(Diana,Rocha)
(Eduarda,Silva)
(Fábio,Matos)
(Francisco,Ferreira)
(Fernando,Teixeira)
(Fabiana,Alves)
(Gisele,Ferreira)
(Gonçalo,Cruz)
(Gustavo,Lima)
(Henrique,Coelho)
(Hugo,Vieira)
(Helena,Monteiro)
(Inês,Mendes)
(Joana,Almeida)
(Júnior,Bastos)
(José,Martins)
(Juliano,Rocha)
(Jéssica,Costa)
(João,Pinto)
(Kelly,Neves)
(Leonor,Ferreira)
(Laura,Gomes)
(Larissa,Amorim)
(Luciana,Nogueira)
(Luís,Carvalho)
(Márcio,Coelho)
(Miguel,Pires)
(Marta,Ribeiro)
(Marco,Ferreira)
(Margarida,Maggie)
(Mariana,Matos)
(Maria,Costa)
(Nicolas,Mendes)
(Patrícia,Silva)
(Pedro,Santos)
(Paulo,Carvalho)
(Raquel,Moura)
(Rodrigo,Azevedo)
(Rogério,Lima)
(Rui,Nogueira)
(Rafael,Antunes)
(Renata,Teixeira)
(Ricardo,Mendes)
(Sara,Marques)
(Sofia,Monteiro)
(Sérgio,Oliveira)
(Tomás,Barbosa)
(Tatiana,Vieira)
(Tiago,Pereira)
(Viviane,Lima)
(Anderson,Moura)
(Alexandra,Barros)
(André,Monteiro)
(Amanda,Santos)
(Alexandre,Rodrigues)
(Ana,Martins)
(Adriana,Lopes)
(Alberto,Pereira)
(Bruno,Neves)
(Beatriz,Moreira)
(Catarina,Castro)
(Carolina,Bastos)
(Cláudia,Barbosa)
(Carla,Lopes)
(Camila,Ribeiro)
(Clara,Sousa)
(Cristina,Sousa)
(Carlos,Ramos)
(Daniela,Lima)
(Douglas,Ramos)
(Diego,Marques)
(Diana,Rocha)
(Eduarda,Silva)
(Fábio,Matos)
(Francisco,Ferreira)
(Fernando,Pessoa)
(Fabiana,Alves)
(Gisele,Ferreira)
(Gonçalo,Cruz)
(Gustavo,Lima)
(Henrique,Coelho)
(Hugo,Vieira)
(Helena,Monteiro)
(Inês,Mendes)
(Joana,Almeida)
(Júnior,Bastos)
(José,Martins)
(Juliano,Rocha)
(Jéssica,Costa)
(João,Pinto)
(Kelly,Neves)
(Liam,Nelson)
(Leonor,Ferreira)
(Laura,Gomes)
(Larissa,Amorim)
(Luciana,Nogueira)
(Luís,Carvalho)
(Márcio,Coelho)
(Miguel,Pires)
(Marta,Ribeiro)
(Marco,Ferreira)
(Margarida,Maggie)
(Mariana,Matos)
(Maria,Costa)
(Nicolas,Mendes)
(Patrícia,Silva)
(Pedro,Santos)
(Paulo,Carvalho)
(Raquel,Moura)
(Rodrigo,Azevedo)
(Rogério,Lima)
(Rui,Nogueira)
(Rafael,Antunes)
(Renata,Teixeira)
(Ricardo,Mendes)
(Sophia,Andresen)
(Sara,Marques)
(Sofia,Monteiro)
(Sérgio,Oliveira)
(Tomás,Barbosa)
(Tatiana,Vieira)
(Tiago,Pereira)
(Viviane,Lima)
[(Natalie,KVSMISSING)(Oscar,KVSMISSING)]
(Anderson,Moura)
(Alexandra,Barros)
(André,Monteiro)
(Amanda,Santos)
(Alexandre,Rodrigues)
(Ana,Martins)
(Adriana,Lopes)
(Alberto,Pereira)
(Bruno,Neves)
(Beatriz,Moreira)
(Chris,Hemsworth)
(Catarina,Castro)
(Carolina,Bastos)
(Cláudia,Barbosa)
(Carla,Lopes)
(Camila,Ribeiro)
(Clara,Sousa)
(Cristina,Sousa)
(Carlos,Ramos)
(Daniela,Lima)
(Douglas,Ramos)
(Diego,Marques)
(Diana,Rocha)
(Emma,Stone)
(Eduarda,Silva)
(Fábio,Matos)
(Francisco,Ferreira)
(Fernando,Pessoa)
(Fabiana,Alves)
(Gisele,Ferreira)
(Gonçalo,Cruz)
(Gustavo,Lima)
(Henrique,Coelho)
(Hugo,Vieira)
(Helena,Monteiro)
(Inês,Mendes)
(Jorge,Luis)
(Joana,Almeida)
(Júnior,Bastos)
(José,Martins)
(Juliano,Rocha)
(Jéssica,Costa)
(João,Pinto)
(Kelly,Neves)
(Liam,Nelson)
(Leonor,Ferreira)
(Laura,Gomes)
(Larissa,Amorim)
(Luciana,Nogueira)
(Luís,Camões)
(Márcio,Coelho)
(Miguel,Cervantes)
(Marta,Ribeiro)
(Marco,Ferreira)
(Margarida,Maggie)
(Mariana,Mari)
(Maria,Costa)
(Nicolas,Mendes)
(Pablo,Neruda)
(Patrícia,Silva)
(Pedro,Santos)
(Paulo,Carvalho)
(Raquel,Moura)
(Rodrigo,Azevedo)
(Rogério,Lima)
(Rui,Nogueira)
(Rafael,Antunes)
(Renata,Teixeira)
(Ricardo,Mendes)
(Scarlett,Johansson)
(Sophia,Andresen)
(Sara,Marques)
(Sofia,Monteiro)
(Sérgio,Oliveira)
(Tomás,Barbosa)
(Tatiana,Vieira)
(Tiago,Pereira)
(Viviane,Lima)
